Automattic/woocommerce-payments
# Track overview balance

Fixes https://github.com/Automattic/woocommerce-payments/issues/5880

#### Changes proposed in this Pull Request

- Track when user selects currency tab by click or by keyboard `Enter` key.
- Track when user opens the balance tooltips by click or by keyboard `Enter` key.

ps: The issue https://github.com/Automattic/woocommerce-payments/issues/5880 mentions to track when tooltip is closed but I think it's not too important to track it, so I do not include it in this PR. Let me know if I should.

#### Testing instructions

* Make sure wp-admin > WooCommerce > Settings > Advanced > WooCommerce.com > `Enable tracking` is on.

* Select a currency tab by clicking or by navigating using the `tab` key and hit the `Enter`.
* Expect a request to `t.gif` with payload consisting of `_en: wcadmin_wcpay_overview_balance_currency_click` and `currency: {currency}`.

* Open the balance tooltips by clicking or by navigating using the `tab` key and hit the `Enter` key the currency tab.
* Expect a request to `t.gif` with payload consisting of `_en: wcadmin_wcpay_overview_balance_tooltip_click ` and `context: {tooltip label}`.

-------------------


**Post merge**
